src/app/pages/home-page/components/text-image/text-image.component.html

```diff
@@ -1,4 +1,4 @@
-<div class='container' [ngClass]="{ right: position === 'right', left: position === 'left' }">
+<div class='container row justify-content-center align-items-center' [ngClass]="{ right: position === 'right', left: position === 'left' }">
```

Comment:
row, justify-content-center and align-items-center all have few CSS properties. Can we please make one CSS class with all of the and assign it to this div?